tests/unit/test_speechManager/__init__.py

```diff
@@ -872,12 +869,12 @@ def test_4_profiles(self):
 			smi.expect_synthSpeak(sequence=[
 				seq[1],  # PitchCommand
 				'9 10 11 12',
-				smi.create_ExpectedIndex(expectedToBecomeIndex=4)
+				ExpectedIndex(expectedIndexCommandIndex=3)

```

Comment:
There is still this one that could be `smi.create_ExpectedIndex` I believe. 
```suggestion
				smi.create_ExpectedIndex(expectedToBecomeIndex=3)
```